docs/devctr-image.md

```diff
@@ -46,9 +46,31 @@ registry. The Firecracker CI suite must also be updated to use the new image.
 
 1. Build a new container image with the updated Dockerfile.
 
+   The below script is used to build a docker image.
+
    ```bash
-    docker build -t fcuvm -f tools/devctr/Dockerfile.x86_64 .
-    ```
+   tools/devtool build_devctr
+   ```
+
+   The script uses two files -
+
+    a) ```requirements.txt``` - This contains list of python packages
+    with their unbounded versions.
+
+    b) {arch-based}-requirements-freeze.txt
+    (eg ```x86-requirements-freeze.txt```) - This contains exhaustive
+    list of python dependencies based on dependencies defined in
+    requirements.txt. Unlike requirements.txt, package
+    version in this file are bounded.
+
+   #### How does requirements-freeze.txt gets updated?
+
+   In every run the script finds if new packages are added in
+   requirements.txt and adds newly added packages in
+   {arch-based}-requirements-freeze.txt.
+
+   The script also looks for all outdated packages on the container,
```

Comment:
"The script also looks for any outdated packages in the container and prompts the user to upgrade them"